### ISSUE TITLE
replies to myself are listen under "toots" xand not under "toots and replies"

### ISSUE THREAD

**Issue by schiessle** [tootsuite/mastodon#8751]
I just noticed that if I answer to my own toot to add something which didn't fit in the first 500 chars, the reply is listed in the "toots" column of the web interface. I would expect to see it in the "toots and replies" column only. Having it in the "toots" list makes it confusing for two reasons:

1. The reply comes before the actual toot, it is not marked as reply and there is no easy way to know that it is reply
2. It is quite confusing to look at a users list of toots and see toots which looks strange with missing context, e.g. if the reply is something along the line "and BTW, that's the link to it http...."

**Comment by schiessle** [comment]
> You do not respond to yourself, like a Q&A or conversation -- you continue your thoughts.

I completely agree with this assumption. That's exactly why I find it wired to see the continuation of a thought in my timeline and even above the first part. So people looking at my timeline see the second part out of context which might even make no sense at all standing by it's own. As a continuation it should be always (directly) below the first part in order to have the right context and nested accordingly so that it is always clear what belongs to what.

**Comment by nightpool** [comment]
Mastodon doesn't thread posts in-timeline, it displays all posts in a
reverse chronological order

**Comment by schiessle** [comment]
> Mastodon doesn't thread posts in-timeline, it displays all posts in a reverse chronological order

O course you are right. Mastodon doesn't have nested threads but I think you got what I meant. :wink:
